### PR TITLE
[LUNE] - Store page

### PR DESCRIPTION
**Pull request by ciampo**
This a PR for the store page of the Lune template.

Mocks can be found in /mocks:
- Narrow - Store.jpg
- Wide - Store.jpg

### CODE REVIEW

File: templates/lune/store.amp.html [hunk 1]
```diff
@@ -17,9 +17,73 @@
...
     {{> ../../components/navbar/navbar.snip.html}}
 
+    <!--
+      To enable the parallax effect on the image, edit store.amp.json:
+        1. Add "amp-fx-parallax" to the list of the extensions
+        2. Change the value of "parallax-amount" to a number (e.g. 0.7)
+    -->
+
+    <!-- STORE PAGE -->
+    {{#store-page}}
```
Comment (camelburrito) [inline templates/lune/store.amp.html]: The only issue i have with this PR is the semantics and nested divs 
I left a comment on how you could solve it in your previous PR. Rest if it looks fine.

**Comment by ciampo** [comment]
Hi @camelburrito ,

I made made some amends to the markup, trying to use more semantic tags and to remove some nested nodes where possible. I also applied some of these changes to the landing page too.

File: templates/lune/store.amp.html (hunk 1 above)
Comment (ciampo) [inline templates/lune/store.amp.html]: I made a few amends, let me know if they work for you

**Comment by ciampo** [comment]
Hey @camelburrito,
is there anything else you'd like me to change in this PR that prevents it from being merged?

**Pull request merged**